 
New instance of bashtop version: 0.9.25 Pid: 3737
 
New instance of bashtop version: 0.9.25 Pid: 17542
/usr/bin/bashtop: line 4806: ${pycoproc[1]}: ambiguous redirect
 
New instance of bashtop version: 0.9.25 Pid: 3579
 
New instance of bashtop version: 0.9.25 Pid: 5860
 
New instance of bashtop version: 0.9.25 Pid: 6771
 
New instance of bashtop version: 0.9.25 Pid: 7499
Traceback (most recent call last):
  File "/tmp/iE7G3N3s3uCm/bashtop.psutil", line 395, in <module>
    exec(command)
  File "<string>", line 1, in <module>
  File "/tmp/iE7G3N3s3uCm/bashtop.psutil", line 224, in get_proc
    for p in sorted(psutil.process_iter(['pid', 'name', 'cmdline', 'num_threads', 'username', 'memory_percent', 'cpu_percent', 'cpu_times', 'create_time'], err), key=lambda p: eval(sort_cmd), reverse=reverse):
  File "/usr/lib/python3.9/site-packages/psutil/__init__.py", line 1447, in process_iter
    yield proc
KeyboardInterrupt
